src/main/java/net/pms/formats/v2/AudioProperties.java

```diff
@@ -83,6 +116,33 @@ public void setNumberOfChannels(String mediaInfoValue) {
 	}
 
 	/**
+	 * Get bits per sample for this audio track.
+	 * @return bits per sample (default is 16)
+	 */
+	public int getBitsperSample() {
+		return bitsperSample;
+	}
+
+	/**
+	 * Set bits per sample for this audio track.
+	 * @param bitsperSample bits per sample to set
+	 */
+	public void setBitsperSample(int bitsperSample) {
+		if (bitsperSample < 1) {
+			throw new IllegalArgumentException("Bits per sample can't be less than 1.");
+		}
```

Comment:
OK, i will revert it because i think keep the tests is more important, and i don't know how to fix the tests so...

That's said i share your view on that, but i'm only a patchy guy, so i avoid dreaming about catching the moon.